### DISCUSSION TITLE
Horrible results from the model trained on particular subject / photo

### DISCUSSION THREAD

**Discussion by Tikonderoga** [TheLastBen/fast-stable-diffusion#1088] [Q&A]
Hello.

No matter how many photos of person I'm using, no matter how close they are to each other (series of shots all done the same day, of mix of some recent photos with some older ones), I'm always getting txt2img pictures from the trained model that looks like this:

![trained_model_3000steps](https://user-images.githubusercontent.com/36517077/208491145-7514d59b-d8b2-4f0e-b335-c22a48f10479.jpg)

I'm using default settings (1.5 base model), 3000 training steps, 350 steps for textual inversion.
I strongly suspect that somethings is horribly wrong on my side, but I have no clue what.

Any advice would be appreciated, thanks!

**Comment by Tikonderoga** [comment]
Interestingly enough, if I'm trying to add more "artistic" tags to the prompts, these cadavers become more lifelike and even resemble the original person a bit. But any attempt to get "close up photo of <tag> person on the small town street" are horrendous.

**Reply by TheLastBen** [reply]
what did you name the instance images ?

**Reply by Tikonderoga** [reply]
I'm using format "tag (Number).jpg" format, where tag is "dakat".

**Reply by TheLastBen** [reply]
how many instance images and are they high resolution clear pictures ?

**Reply by Tikonderoga** [reply]
I've tried from 9 to 30 images. Original photos are from phone camera, JPEGs converted to PNG format, cropped to 1x1 ratio squares and resized to 512x512.

**Comment by nawnie** [comment]
would you mind trying a diffrent person, or sharing the instance images, if another subject doesnt train.. well it may be something on your end, id it does, its possible its your subject or instance images, i had a hard time training my face on most DB.

**Comment by SorenTruelsen** [comment]
It looks a bit like overtraining, do you get the same results from the intermediate CPKT's? 

Try run an X/Y with X = steps (20, 40, 60) and Y = All your intermediate checkpoints. And the prompt something like "Face of dakat" and put 3 in each batch.

Also, it could be the tag "dakat" - If you do another training, pick something gibberish, like "fgfgsdsf". I had problems with a person I called "Deamon", but when I changed it into DFSTR it improved. Could be coincident, who knows.  The first 4 letters are very predictive.
